Packs/DelineaSS/Integrations/DelineaSS/DelineaSS.py

```diff
@@ -478,8 +478,28 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newpassword:
     )
 
 
+def get_credentials(client,secret_id):
+    obj={}
+    secret = client.getSecret(secret_id)
+    items = secret.get('items')
+    for item in items:
+        if item.get('fieldName') == 'Username':
+            username = item.get('itemValue')
+        if item.get('fieldName') == 'Password':
+            password = item.get('itemValue')
+            obj = {
+                "user": username,
+                "password": password,
+                "name": str(secret.get('id'))
+            }

```

Comment:
Why does `obj` configured only if item is password? i.e., the obj definition is intended under `if item.get('fieldName') == 'Password'` so user_name will never have a value in obj.

Did you mean this?
```suggestion
        if item.get('fieldName') == 'Password':
            password = item.get('itemValue')
        obj = {
            "user": username,
            "password": password,
            "name": str(secret.get('id'))
        }

```

Reply:
We want to save all the data in obj. I move the obj as you suggested it shows the error for the username is referenced before the assignment. Current implementation is working fine